### PR TITLE
Generate collection thumbnail

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
https://www.loom.com/share/df2bc6a33c38487d956e05a9eb552353
DeepinScreenshot_select-area_20201002012410

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/7512

### CODE REVIEW

File: app/grants/views.py [hunk 1]
```diff
@@ -2247,3 +2252,18 @@ def add_grant_from_collection(request, collection_id):
     return JsonResponse({
         'grants': grants,
     })
+
+
+@login_required
+@staff_member_required
```
Comment (thelostone-mc) [inline app/grants/views.py:2258]: ^ why does need a login + staff decorator?

Comment (zoek1) [reply app/grants/views.py:2258]: This Is for development and debug purposes like the email views.

**Comment by thelostone-mc** [comment]
@zoek1 could you fix up the conflicts as well

**Comment by zoek1** [comment]
conflicts fixed @thelostone-mc

File: app/grants/views.py [hunk 2]
```diff
@@ -2277,3 +2282,18 @@ def add_grant_from_collection(request, collection_id):
...
+def collection_thumbnail(request, collection_id):
+    width = int(request.GET.get('w', 600))
+    height = int(request.GET.get('h', 400))
+    collection = GrantCollection.objects.get(pk=collection_id)
```
Comment (thelostone-mc) [inline app/grants/views.py:2292]: ^ try + catch ?

File: app/grants/templates/grants/components/collection.html [hunk 3]
```diff
@@ -33,9 +33,10 @@
               <img :src="getGrantLogo(3)" alt="" v-if="collection.grants.length >= 4">
             </div>
           </div>
-          <div class="collection-item__owner-image--cover">
+          <div class="collection-item__owner-image--cover" v-if="!collection.cover">
             <img :src="collection.owner.avatar_url || 'https://c.gitcoin.co/avatars/57e79c0ae763bb27095f6b265a1a8bf3/thelostone-mc.svg'">
```
Comment (octavioamu) [inline app/grants/templates/grants/components/collection.html:37]: wait this is from the old code but if not user is default to thelostone avatar? users always have an avatar_url, if not the fallback should be github one, the same way we do in other places

**Review by octavioamu — CHANGES_REQUESTED** [review]
looking good just left a comment about a default user avatar

**Review by PixelantDesign — CHANGES_REQUESTED** [review]
Could we make the dark border thinner and put a circle behind the avatar, and make it 30% larger? 

See example here

Screen Shot 2020-10-13 at 1 23 21 AM

**Comment by zoek1** [comment]
what do you think @PixelantDesign? 
image

File: app/grants/views.py (hunk 2 above)
Comment (zoek1) [reply app/grants/views.py:2564]: IMHO since this method is for development purposes is better to show the exceptions instead of caching them.

**Review by chibie — APPROVED** [review]

**Pull request merged**